### ISSUE TITLE
[Question] The matched versions of apisix and apisix-dashboard projects

### ISSUE THREAD

**Issue by imjoey** [apache/incubator-apisix-dashboard#292]
Hi all, first I would thank you for this awesome project. I'm on my way of running the apisix v1.4 along with its dashboard and everything works fine. While, if I take the latest dashboard code from `master` branch, I always encounter the error prompt message:  **请求错误，错误码： 401  用户没有权限（令牌、用户名、密码错误）。** 

After some digs, I found that there was no `api-key` header in the request. So I was wondering which version of apisix should I use for the latest dashboard code?

Many thanks.

**Comment by juzhiyuan** [comment]
Hi, the new Dashboard is come up with Docker Compose, Please follow the instructions on the README file under branch master.

**Comment by imjoey** [comment]
Hi @juzhiyuan, sorry for not getting it before and thank you very much for the hint.  In addition, any schedule for adding support for independent binary deployment? Thanks.

**Comment by juzhiyuan** [comment]
binary deployment? or dashboard without Docker?

**Comment by imjoey** [comment]
Yeap, non-containerized deployment.

**Comment by juzhiyuan** [comment]
I have no idea because currently, we need the Docker Compose to set up some APIs (we called manager-api), which is as a bridge between Dashboard and APISIX.

**Comment by juzhiyuan** [comment]
The manager API handles some logics which should not be handled by APISIX.

**Comment by imjoey** [comment]
> The manager API handles some logics which should not be handled by APISIX.

Hi @juzhiyuan , much appreciated to your reply. I guess I could find how to deploy without containerization by digging the docker compose yaml files.

Regarding to the versions, I found that `apache/apisix:dev`, which is built via the latest codes in master branch of `apache/apisix`. So I've got what I want. Thanks again for your help. Let's close this.
